webapp/tests/Controller/Jury/ContestControllerTest.php

```diff
@@ -0,0 +1,19 @@
+<?php declare(strict_types=1);
+
+namespace App\Tests\Controller\Jury;
+
+use App\Entity\Contest;
+
+class ContestControllerTest extends JuryControllerTest
+{
+    protected static $baseUrl         = '/jury/contests';
+    protected static $shortTag        = 'contest';
+    protected static $deleteEntities  = array('name' => ['Demo practice session']);
```

Comment:
Please use short array syntax (`[]`)

Reply:
I'm under the impression that my editor is in PSR12 mode.. will re-check.

Reply:
done